stubs/enterReplies.js

```diff
@@ -11,10 +11,17 @@ icebreakers = [
   'When was the last time you changed your hairstyle?',
   'Have you committed today?',
   'You\'re trapped on a deserted island, what TV show are you bringing to keep you company?',
-  '' 
+  'What\'s your favorite flavor of sparkling water?',
+  'Tell us your favorite \'nerdy\' joke',
+  'How old were you when you stopped believing in Santa Claus?',
+  'Share your favorite color (in hexcode!)',
+  'What website do you suddenly find yourself on when you\'re \"just checking your email\"?',
+  'If you could be any other species, which would you choose?',
+  'What superpower would you want?',
+  'What\'s your patronus?'
 ];
 
 // Combine introduction with an icebreaker question
-responses = icebreakers.map((item) => introduction + ' ' + item) + 'Start a thread with your response to get to know the hack lab better';
+responses = icebreakers.map((item) => introduction + ' ' + item + 'Start a thread with your response to get to know the hack lab better');
```

Comment:
I'm not sure what `item` returns but should there be a space in there? `...item + ' Start...`

Reply:
Thanks Kevin! Yes, there does probably need to be a space or newline in there. I'll work with Pat getting the test bot going so I can see the output and get it fined tuned.